agent/agent.go

```diff
@@ -84,6 +85,7 @@ func NewAgent(ctx context.Context, cancelF context.CancelFunc) (*Agent, error) {
 		sandboxed: isSandboxed(),
 		md:        metadata,
 		started:   time.Now().UTC(),
+		subz:      make([]*nats.Subscription, 0),
```

Comment:
I still maintain that manually and explicitly unsubscribing every subscription isn't/shouldn't be necessary, but as long as it doesn't actually make things worse, I'm okay with it.